src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    public static void hi() {
```

Comment:
The name of the method have to be a verb instead:
https://se-education.org/guides/conventions/java/basic.html#naming